README.md

```diff
@@ -15,6 +15,13 @@ https://owdocs.vercel.app
 5. `pnpm dev` to run the project
    - You can run `pnpm dev --filter=<project>` to run a certain project from root, such as `pnpm dev --filter=web`
 
+## Environment
+You need to set up environment variables for the project to work. You can get them on [[doppler.com]], or using the cli:
```

Comment:
does this work as a markdown link? :o

Reply:
works in my vscode but apparently it's non-standard, fixed./